spyder/api/widgets/comboboxes.py

```diff
@@ -255,10 +378,24 @@ def __init__(self, parent=None):
 class SpyderFontComboBox(QFontComboBox, _SpyderComboBoxMixin):
 
     def __init__(self, parent=None):
-        super().__init__(parent)
+        if PYQT5 or PYQT6:
+            super().__init__(parent)
+        else:
+            QFontComboBox.__init__(self, parent)
+            _SpyderComboBoxMixin.__init__(self)
```

Comment:
This is necessary for PySide2 because it has issues with Python's MRO when a class inherits from several other classes (see PR #16322).

But it won't be necessary for PySide6 because it seems those problems were finally solved in its more recent versions (6.7+, I think).